modules/apps/headless/headless-delivery/headless-delivery-api/src/main/java/com/liferay/headless/delivery/dto/v1_0/util/ContentFieldUtil.java

```diff
@@ -114,14 +114,13 @@ public static ContentField toContentField(
 						}
 
 						Map<String, ContentFieldValue> map = new HashMap<>();
+						Value value = ddmFormFieldValue.getValue();
 
-						Map<Locale, String> valueValues = Optional.ofNullable(
-							ddmFormFieldValue.getValue()
-						).map(
-							Value::getValues
-						).orElse(
-							Collections.emptyMap()
-						);
+						Map<Locale, String> valueValues = new HashMap<>();
```

Comment:
@jeffwu0724 Originally it returns `Collections.emtpyMap()` as fallback, please be consistent.

Also, please try to refine the logic and naming here.